### ISSUE TITLE
Marketplaces (positive interoperability) placeholder

### ISSUE THREAD

**Issue by alltheseas** [nostrability/nostrability#65]

**Comment by alltheseas** [comment]
https://github.com/nostr-protocol/nips/blob/master/15.md

Gzuus to update

**Comment by fiatjaf** [comment]
This is a thing I don't know much about, we should tag everybody doing market stuff here so we can finally discover what is going on.

**Comment by alltheseas** [comment]
@gzuuus is working on an yet to be shared spec via plebeian market, and will share here when ready.

We'll then get @vitorpamplona calvadev and co to review

**Comment by vitorpamplona** [comment]
Are we talking about Marketplaces (NIP-15) or Classified Listings (NIP-99). NIP-15 is for online stores, NIP-99 is for one-off listings like Facebook marketplace

**Comment by gzuuus** [comment]
Yes, I was chatting with @calvadev during BH2024, and if I'm not mistaken, we are the only ones actively working on implementing e-commerce over Nostr. The issue is that NIP-15 and NIP-99 overlap, making it unclear what unique use case each serves. Furthermore, NIP-99 only describes listing, without outlining an end-to-end experience, which may create an inconsistent interoperability experience for users who try to use it across a (hypothetical) richer ecosystem of ecommerce apps. During our conversation, we brainstormed different ideas and agreed to develop this further during SEC-03, with the goal of refining and updating the spec to provide a more consistent and robust e-commerce experience over Nostr.

**Comment by staab** [comment]
Coracle supports listings, including listings posted to nip 72 communities and nip 87 groups.

**Comment by alltheseas** [comment]
Today shopstr and amethyst interopwrate

https://njump.me/note1w405cqjglqevv2c3420kf95e5tmag8fg05fllxq0yagk3km776hs3uzhvk

@calvadev @vitorpamplona is the current implementation nip-99 for yall?

**Comment by calvadev** [comment]
Yes! NIP-99 is the spec used atm.

**Comment by alltheseas** [comment]
@calvadev @gzuuus any update on spec convergence on marketplaces :eyes:

**Comment by calvadev** [comment]
> @calvadev @gzuuus any update on spec convergence on marketplaces 👀

Working through it this week!

**Comment by alltheseas** [comment]
Latest draft: https://github.com/gzuuus/nips/blob/working-branch/XX.md

**Comment by alltheseas** [comment]
@erskingardner advises the marketplace spec will not converge.

What do yall say @gzuuus @calvadev 

https://njump.me/note1l59yjpdd7wq9vwpdvtv7cln8fe6nd8vkv6kaxjpckw4n2d40spespdd6nr

**Comment by erskingardner** [comment]
My note wasn't a pronouncement that anyone was doing anything wrong or that either NIP shouldn't be extended. It was just a statement that the original marketplace NIP (NIP-15) wasn't likely to converge with NIP-99 Classifieds - they're very different.

**Comment by alltheseas** [comment]
no worries @erskingardner looks like we were talking about different things. this crew should update marketplace NIP used by Shopstr, Plebian Market etc in due course

**Comment by alltheseas** [comment]
Www.Resin.estate uses nip-99

Https://njump.me/note1wgjpad69xmj4fzn0fkvg626q0wg24evgc7u938q8s4g337f26axsrqs6us

**Comment by alltheseas** [comment]
There exists Islamic marketplace

npub1kppw9a5asyvz4c35d0kh8gzulvlealcyw28u0f2mau8n6auna84sdyf9y3

**Comment by alltheseas** [comment]
> There exists Islamic marketplace
> 
> npub1kppw9a5asyvz4c35d0kh8gzulvlealcyw28u0f2mau8n6auna84sdyf9y3

> No actual sales or payments take place there. It's just a directory for making contact. Everything else is handled directly between the customer and the provider.

https://njump.me/note1szmzdcdhlz3kysg24vpjnh88vw3ssanceuce0l6ywyvz3tpgyg6sdyhw5u

**Comment by alltheseas** [comment]
https://github.com/nostr-protocol/nips/pull/1703/commits/efe06d9f2b80170e63df18d1b6ed81b2b1b0a9df

**Comment by alltheseas** [comment]
@mmalmi fyi placeholder unified marketplace tracking issue

**Comment by alltheseas** [comment]
Latest spec: https://github.com/GammaMarkets/market-spec/blob/main/spec.md

**Comment by alltheseas** [comment]
See NIP-99

https://github.com/nostr-protocol/nips/blob/master/99.md

**Comment by alltheseas** [comment]
Wonderful demo of marketplace interoperability across three apps:

https://njump.me/nevent1qqs2wy4qkuutdwuetgt7qeaqknqq44v4f7xewrxuwwhhstsvcd68ncgwz5gt5

**Comment by alltheseas** [comment]
@mmalmi @vitorpamplona are y'all on latest [NIP-99](https://github.com/nostr-protocol/nips/blob/master/99.md) marketplace spec for Iris, and Amethyst?

**Comment by vitorpamplona** [comment]
No, I don't think so. I haven't touched that part of the code in over 2 years. :(
